index.html

```diff
@@ -0,0 +1,85 @@
+        <h1>Hi,I am Mimic</h1>
+        <p>a Web Developer</p>
+      </div>
+      <section id="projects">
+        <div class="project-tile">
+       <h2>These are some of my projects</h2>
+        </div>
+        <div id="project-link">
+         <div id="image-link">
+          <a  href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank"><img id="image" src="images/download.jpg" alt=""></a>
+          <a id="link" href="https://codepen.io/freeCodeCamp/full/zNqgVx"  target="_blank" >Tribute</a>
+         </div>
+         <div id="image-link">
+            <a  href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank"><img id="image" src="images/download.jpg" alt=""></a>
+            <a id="link" href="https://codepen.io/freeCodeCamp/full/zNqgVx"  target="_blank" >Tribute</a>
+           </div>
+           <div id="image-link">
+            <a  href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank"><img id="image" src="images/download.jpg" alt=""></a>
+            <a id="link" href="https://codepen.io/freeCodeCamp/full/zNqgVx"  target="_blank" >Tribute</a>
+           </div>
+           <div id="image-link">
+            <a  href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank"><img id="image" src="images/download.jpg" alt=""></a>
+            <a id="link" href="https://codepen.io/freeCodeCamp/full/zNqgVx"  target="_blank" >Tribute</a>
+           </div>
+           <div id="image-link">
+            <a  href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank"><img id="image" src="images/download.jpg" alt=""></a>
+            <a id="link" href="https://codepen.io/freeCodeCamp/full/zNqgVx"  target="_blank" >Tribute</a>
+           </div>
+           <div id="image-link">
+            <a  href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank"><img id="image" src="images/download.jpg" alt=""></a>
+            <a id="link" href="https://codepen.io/freeCodeCamp/full/zNqgVx"  target="_blank" >Tribute</a>
+           </div>
+        </div>
+        <div id="button">
+            <a  href="https://codepen.io/FreeCodeCamp/" target="_blank"> <button type="button" class="btn btn-lg btn-dark">Show all</button> </a>
+           </div>
+      </section>
+      <footer id="footer">
+        <h1>Let's work together ..</h1>
+        <div id="footer-detail">
+            <p>How do you take your coffee?</p>
+        </div>
+        <div id="icons">
+            <a href="https://www.facebook.com/freecodecamp" target="_blank"><i class="fa-brands fa-facebook"></i>Facebook</a>
+            <a href="https://github.com/freecodecamp" target="_blank"><i class="fa-brands fa-github"></i>GitHub</a>
+            <a href="https://twitter.com/freecodecamp" target="_blank"><i class="fa-brands fa-twitter"></i>Twitter</a>
+            <a href="" target=""><i class="fa-solid fa-at"></i>Send me a mail</a>
+            <a href="" target=""><i class="fa-sharp fa-solid fa-mobile"></i>Call me</a>
```

Comment:
Lovely work 👏🏻 on the container of the icons, however, I suggest you please use ul instead of div because using ul over the div is semantically more correct. It would help your website to be more accessible to the audiences that have to rely on the screen-reader. Kindly make sure that all your child list in the icons also use li to make it perfect otherwise it will results again in bad practices.